src/ServiceControl.Transports.ASQ/Helper.cs

```diff
@@ -16,11 +13,6 @@ public static void ApplyHacksForNsbRaw(this TransportExtensions<AzureStorageQueu
             var settings = extensions.GetSettings();
             var serializer = Tuple.Create(new NewtonsoftSerializer() as SerializationDefinition, new SettingsHolder());
             settings.Set("MainSerializer", serializer);
-
-            var ctor = typeof(MessageMetadataRegistry).GetConstructor(BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Instance, null, new[] {typeof(Func<Type, bool>)}, null);
```

Comment:
Pretty sure this is no longer needed because of this change

https://github.com/Particular/NServiceBus.AzureStorageQueues/pull/304

Reply:
If the tests pass then I am fine with this change ;-)